neo/Helper.cs

```diff
@@ -140,11 +140,13 @@ internal static bool TestBit(this BigInteger i, int index)
             return (i & (BigInteger.One << index)) > BigInteger.Zero;
         }
 
+        // TODO - WHO USES THIS?
```

Comment:
@erikzhang

Reply:
@vncoelho  remove it and you will find out 😄 
I'm in favor of removing "dead" code. If it runs without it, we can delete it.